solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -1903,21 +1922,20 @@ private ZkCoreNodeProps waitForLeaderToSeeDownState(
             " is already less than leader, so not waiting for leader to see down state.");
       } else {
 
-        log.info("Replica need to wait for leader to see down state.");
+        log.info("Replica should to wait for leader to see down state.");
```

Comment:
Replica `is` waiting ?

Reply:
Also, while you are at it, can you add more information e.g. specifying the replica ?